src/managers/nong_manager.cpp

```diff
@@ -205,7 +205,7 @@ void NongManager::deleteNong(SongInfo const& song, int songID) {
             if (song.path == existingData.active) {
                 existingData.active = existingData.defaultPath;
             }
-            if (song.songUrl != "local" && existingData.defaultPath != song.path && fs::exists(song.path)) {
+            if (deleteFile && song.songUrl == "local" && existingData.defaultPath != song.path && fs::exists(song.path)) {
```

Comment:
I think you can delete the == "local" check from here. It isn't really used anymore